examples/can-receive/index.rsh

```diff
@@ -0,0 +1,47 @@
+'reach 0.1'
+
+export const main = Reach.App(() => {
+  const P = {
+    seeCanReceive: Fun([Token, Bool], Null),
+    seeInt: Fun([UInt], Null),
+  }
+  const Alice = Participant('Alice', {
+    ...P,
+  })
+  const Bob = Participant('Bob', {
+    ...P,
+  })
+  init()
+
+
+  Alice.publish()
+  const tok = new Token({supply: 10, decimals: 0})
+  commit()
+
+  Bob.publish()
+  // Use canReceive in a consensus step.
+  const x = canReceive(Bob, tok) ? 1 : 0
```

Comment:
`x` is not used, so it is probably being optimized away. And you should name this function `Token.accepted(acc, tok)` or `Account.tokenAccept(acc, tok)` [or both] not `canReceive`

Reply:
`x` is used by an interact function.  Can that still be optimized away?  FYI I named it `canReceive` because that's what you wrote in the Jira card, but I agree that the other options are better (they better match the stdlib function).

Reply:
It has to be used in a consensus step. For example, you could have `Bob.publish(z)` and then do `check(x == z)` and it will require `x` to be computed and saved